### PR TITLE
seeds updated and good to go

### PR DESCRIPTION
**Pull request by mkewls**
** 10 Plans
** 20 Treatments
** 50 Exercises
** 100 Workouts

### CODE REVIEW

File: db/Seed/planSeed.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "duration": 14,
+    "therapy_focus": "stable",
+    "notes": "Morbi vestibulum, velit id pretium iaculis, diam erat fermentum justo, nec condimentum neque sapien placerat ante.",
+    "patient_id": 1
+  },
+  {
+    "duration": 19,
+    "therapy_focus": "conglomeration",
```
Comment (jakepeyser) [inline db/Seed/planSeed.json]: 😆

File: db/Seed/workoutSeed.json [hunk 2]
```diff
@@ -0,0 +1,802 @@
+[
```
Comment (mkewls) [inline db/Seed/workoutSeed.json]: Ah, fair point. I wonder if I can override the built-in hook and fake the 'created_at'. Will try.

Comment (mkewls) [inline db/Seed/workoutSeed.json]: OK, so I couldn't force fake data into the Sequelize timestamp hook. Instead, I disabled it and then added a 'created_at' column which I seeded. If that works in your view, this is ready to merge.

Comment (jakepeyser) [inline db/Seed/workoutSeed.json]: Yea, I like that. So we just need to be sure to include that whenever we create a Workout instance

**Pull request merged**